packages/api/src/command/medical/invalid-links/get-invalid-links.ts

```diff
@@ -0,0 +1,37 @@
+import { Transaction } from "sequelize";
+import NotFoundError from "@metriport/core/util/error/not-found";
+import { InvalidLinksModel } from "../../../models/invalid-links";
+
+export type GetInvalidLinks = {
+  id: string;
+  cxId: string;
+  transaction?: Transaction;
+  lock?: boolean;
+};
+
+export async function getInvalidLinks({
+  id,
+  cxId,
+  transaction,
+  lock = false,
+}: GetInvalidLinks): Promise<InvalidLinksModel | undefined> {
+  const invalidLinks = await InvalidLinksModel.findOne({
+    where: { cxId, id },
+    transaction,
+    lock,
+  });
+  return invalidLinks ?? undefined;
+}
+
+export async function getInvalidLinksOrFail({
+  id,
+  cxId,
+  transaction,
+  lock = false,
```

Comment:
Please leave the default value to the `getInvalidLinks` function. The `OrFail` one doesn't use if for anything, so we might as well leave the one that really uses it define that and be the source of truth.